x-pack/platform/plugins/shared/fleet/server/types/models/package_policy.ts

```diff
@@ -426,6 +426,7 @@ export const PackagePolicyResponseSchema = PackagePolicySchema.extends({
       })
     ),
     SimplifiedPackagePolicyInputsSchema,
+    schema.any(),
```

Comment:
this type needs to replaced with the correct one, for now I added any to allow returning the config in the API response.